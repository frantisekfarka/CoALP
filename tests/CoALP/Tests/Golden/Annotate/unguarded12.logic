q(f(X),f(X),X1,X2):-p(t(X1),t(X2),X1,X2).
p(t(X),t(X),X1,X2):-q(f(X1),f(X2),X1,X2).