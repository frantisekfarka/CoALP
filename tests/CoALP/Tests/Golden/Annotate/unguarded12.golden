q(f(V_2i), f(V_2i), V_3i, V_4i, transform-func-1(V_8c)) :- p(t(V_3i), t(V_4i), V_3i, V_4i, V_8c).
p(t(V_5i), t(V_5i), V_6i, V_7i, transform-func-2(V_9c)) :- q(f(V_6i), f(V_7i), V_6i, V_7i, V_9c).
